lib/exandra/connection.ex

```diff
@@ -671,8 +680,36 @@ defmodule Exandra.Connection do
     do: raise(ArgumentError, "constraints are not supported by Exandra")
 
   @impl Ecto.Adapters.SQL.Connection
-  def execute_ddl({_command, %Index{}, _}),
-    do: raise(ArgumentError, "indexes are not supported by Exandra")
+  def execute_ddl({command, %Index{} = index}) when command in [:create, :create_if_not_exists] do
+    unless is_nil(index.prefix),
+      do: raise(ArgumentError, "indexes with prefixes are not supported by Exandra")
+
+    [
+      [
+        "CREATE ",
+        "INDEX ",
+        if_do(command == :create_if_not_exists, "IF NOT EXISTS "),
+        quote_name(index.name),
+        " ON",
+        ?\s,
+        quote_table(index.prefix, index.table),
+        ?(,
+        intersperse_map(index.columns, ", ", &index_expr/1),
+        ?)
+      ]
+    ]
+  end
+
+  @impl Ecto.Adapters.SQL.Connection
+  def execute_ddl({command, %Index{} = index, _mode}) when command in [:drop, :drop_if_exists] do
+    [
+      [
+        "DROP INDEX ",
+        if_do(command == :drop_if_exists, "IF EXISTS "),
+        quote_table(index.prefix, index.name)
```

Comment:
Do we support prefixes on dropping? Can you do `DROP INDEX foo.some_index`? I thought indexes were scoped globally, so I figured you'd have just `index.name` here too and raise if there's an `index.prefix`.

Reply:
> Do we support prefixes on dropping? Can you do `DROP INDEX foo.some_index`? I thought indexes were scoped globally, so I figured you'd have just `index.name` here too and raise if there's an `index.prefix`.

Yea, I'll raise it